### PR TITLE
Task/part3 creation update of attributes

### PR DESCRIPTION
**Pull request by kzangeli**
Part III of #1259 

Practically everything was done during Part I.
Just some error handling was missing (unless I've misunderstood something).

### CODE REVIEW

File: test/functionalTest/cases/1259_new_json_for_v2/put_v2_entities_eid_attrs_attr.test [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+brokerStart CB 0-255
+
+--SHELL--
+
+#
+# 01. PUT /v2/entities/E1/attrs/A1 and see it fail
+# 02. POST /v2/entities, creating E1 with A1
+# 03. PUT /v2/entities/E1/attrs/A1 and see it work
+# 04. GET /v2/entities/E1/attrs/A1
+# 05. PUT /v2/entities/E1/attrs/A1 'old way' - metadatas not inside 'metadata: {}' - see it fail
```
Comment (fgalan) [inline test/functionalTest/cases/1259_new_json_for_v2/put_v2_entities_eid_attrs_attr.test]: "old" and "new" may become obsolete when this .test goes outside our memory "remembering scope". Maybe could use a "surname" to help to remember:

"old way" -> "old way" (pre-#1259)
"new way" -> "new way" (post-#1259)

File: test/functionalTest/cases/1259_new_json_for_v2/put_v2_entities_eid_attrs_attr.test [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+
+echo "02. POST /v2/entities, creating E1 with A1"
+echo "=========================================="
+payload='{
+  "id": "E1",
+  "type": "T1",
+  "A1": "a1"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
```
Comment (fgalan) [inline test/functionalTest/cases/1259_new_json_for_v2/put_v2_entities_eid_attrs_attr.test]: In order make `"A1": "a1"` to work, may `?options=keyValues` need to be used?

Or maybe right support of `?options=keyValues` is planed for a next PR (in which this .test will be adjusted). Just to check that is under your radar...

File: test/functionalTest/cases/1259_new_json_for_v2/put_v2_entities_eid_attrs_attr.test [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+echo
+
+
+echo "06. PUT /v2/entities/E1/attrs/A1 'new way' - metadatas inside 'metadata: {}' - see it work"
+echo "=========================================================================================="
+payload='{
+  "value": 12,
+  "type": "AT1",
+  "metadata": {
+    "m1":   1,
```
Comment (fgalan) [inline test/functionalTest/cases/1259_new_json_for_v2/put_v2_entities_eid_attrs_attr.test]: Remove extra whitespace?

File: test/functionalTest/cases/1259_new_json_for_v2/put_v2_entities_eid_attrs_attr.test (hunk 2 above)
Comment (kzangeli) [inline test/functionalTest/cases/1259_new_json_for_v2/put_v2_entities_eid_attrs_attr.test]: Exactly.
See description of issue #1259:
7. Selector to make the broker accept different input payloads for /v2/entities

This change came in after I started this work, so, I prefer to first finish what I started and after that make the new changes.

NTC (for this PR)

File: test/functionalTest/cases/1259_new_json_for_v2/put_v2_entities_eid_attrs_attr.test (hunk 1 above)
Comment (kzangeli) [inline test/functionalTest/cases/1259_new_json_for_v2/put_v2_entities_eid_attrs_attr.test]: Fixed in d767fd3 (github-editor - no new functest has been issued ...)

File: test/functionalTest/cases/1259_new_json_for_v2/put_v2_entities_eid_attrs_attr.test (hunk 3 above)
Comment (kzangeli) [inline test/functionalTest/cases/1259_new_json_for_v2/put_v2_entities_eid_attrs_attr.test]: Fixed in a4d597a (also github editor)

**Comment by fgalan** [comment]
LGTM

**Pull request merged**